spec/controllers/projects_controller_spec.rb

```diff
@@ -1,13 +1,85 @@
 require 'spec_helper'
 
+RSpec.shared_examples 'denies access' do
+  before { request }
+
+  it 'redirects to the root path' do
+    expect(response).to redirect_to(root_path)
```

Comment:
I am adding this expectation because this is how we handle the `CanCan::AccessDenied` exception in `application_controller.rb`. Please see https://github.com/TheOdinProject/theodinproject/blob/master/app/controllers/application_controller.rb#L6 

Or am I missing something?